pkg/omf/functions/util/omf.check.version.fish

```diff
@@ -0,0 +1,3 @@
+function omf.check.version -a min_version -a current_version
+  test (echo "$min_version"\n"$current_version" | sort -V | head -n1) = "$min_version"
```

Comment:
Oh... Right. So I need to add something like
```fish
uname -a | grep -qEv "os x|bsd"; or echo "goddammit" >&2
```

j/k. alternate version coming in a second